.github/ISSUE_TEMPLATE/mentee-application.yml

```diff
@@ -0,0 +1,42 @@
+name: Mentee application
+description: Apply to take part in the astropy mentoring scheme as a mentee
+title: "Mentee application - <your name>"
+labels: ["application"]
+assignees:
+  - ''
+body:
+  - type: input
+    id: name
+    attributes:
+      label: Name
+      description: Please give your name
+    validations:
+      required: false
+  - type: input
+    id: usernames
+    attributes:
+      label: GitHub/Slack handles
```

Comment:
If you want two inputs here you should have two boxes?

Reply:
Also should we have an email address box?

Reply:
I figure for a lot of people they'll be the same.

Reply:
Do we need the slack username?

Not sure about the email, it's useful to know and could be obscured?

Reply:
I assume slack username will be the name applicants will be best known by, so I think it makes sense to make sure it's clear who the person is. Again, fine if their github and slack names are the same, but sometimes they're not and they're not always obvious derivations of the person's name.

Sure, if there's a way to make the email address only visible to people we want to have it then that's ideal. I'll have another look at the template docs and see if that's something we can do.

Reply:
I've realised that we will obviously already know the applicant's GitHub name so I've changed this to just be their slack handle if it's different.

For email address, I haven't managed to find a way to obscure it, so unless you know how off the top of your head I'm going to leave it out for now and move on so we can start getting eyes on this from astropy folks. Possibly someone will have opinions on the best way to handle this.